### PR TITLE
HPCC-27776 ECLWatch allow adding groups to default Workunit/Filescope OU

### PR DESCRIPTION
**Pull request by jeclrsg**
Signed-off-by: Jeremy Clements <jeremy.clements@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/eclwatch/ShowIndividualPermissionsWidget.js [hunk 1]
```diff
@@ -1,40 +1,74 @@
...
+        },
+
         //  Hitched Actions  ---
         _onRefresh: function (args) {
             this.grid.refresh();
         },
 
+        fixLabels: function () {
+            query(".AddGroupLabel", this.id + "AddGroupForm").forEach(function (node) {
+                console.log(node);
```
Comment (GordonSmith) [inline esp/src/eclwatch/ShowIndividualPermissionsWidget.js:62]: Left over debugging?

**Review by GordonSmith — CHANGES_REQUESTED** [review]

File: esp/src/eclwatch/ShowIndividualPermissionsWidget.js (hunk 1 above)
Comment (jeclrsg) [reply esp/src/eclwatch/ShowIndividualPermissionsWidget.js]: I forgot to revisit, wanted to fix those labels which contained HTML

**Review by GordonSmith — COMMENTED** [review]
Minor:  Might be worthwhile creating a utility function for:
```
"<center>" + context.i18n.AllowRead.split(" ").join("<br />") + "</center>"
```
?

File: esp/src/src/Utility.ts [hunk 2]
```diff
@@ -1092,6 +1092,36 @@ export function format(labelTpl, obj) {
...
+                const word = parts.shift();
+                if (!word) continue;
+                part += word + " ";
+            }
+            temp[i++] = part.trim();
+        }
+        parts = temp;
+    }
+    return parts.join("<br />");
+}
```
Comment (GordonSmith) [inline esp/src/src/Utility.ts:1112]: Was length > 0 ever used?  What does it do?

**Review by GordonSmith — APPROVED** [review]

**Pull request merged**